homeassistant/components/ecobee/climate.py

```diff
@@ -84,13 +92,32 @@
     PRESET_HOLD_INDEFINITE: "indefinite",
 }
 
-SERVICE_SET_FAN_MIN_ON_TIME = "set_fan_min_on_time"
+SERVICE_CREATE_VACATION = "create_vacation"
+SERVICE_DELETE_VACATION = "delete_vacation"
 SERVICE_RESUME_PROGRAM = "resume_program"
+SERVICE_SET_FAN_MIN_ON_TIME = "set_fan_min_on_time"
 
-SET_FAN_MIN_ON_TIME_SCHEMA = vol.Schema(
+CREATE_VACATION_SCHEMA = vol.Schema(
     {
-        vol.Optional(ATTR_ENTITY_ID): cv.entity_ids,
-        vol.Required(ATTR_FAN_MIN_ON_TIME): vol.Coerce(int),
+        vol.Required(ATTR_ENTITY_ID): cv.entity_id,
+        vol.Required(ATTR_VACATION_NAME): cv.string,
+        vol.Required(ATTR_COOL_TEMP): vol.Any(cv.positive_int, float),
+        vol.Required(ATTR_HEAT_TEMP): vol.Any(cv.positive_int, float),
+        vol.Optional(ATTR_START_DATE): cv.string,
```

Comment:
If the date and time options need to be specified together, we should use `vol.Inclusive`, instead of `vol.Optional`.